Dockerfile

```diff
@@ -4,50 +4,58 @@ LABEL org.label-schema.vcs-url="https://github.com/Dwolla/jenkins-agent-docker-n
 
+WORKDIR ${JENKINS_HOME}
+
 COPY jenkins-agent /usr/local/bin/jenkins-agent
 COPY verify.sh /usr/local/bin/verify.sh
+COPY fake-project $JENKINS_HOME/fake-project
 
-RUN apt-get update && \
-    apt-get install -y curl bash git ca-certificates python make rake g++ apt-transport-https ca-certificates && \
-    apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D && \
+# apt-key loop inspired by https://github.com/nodejs/docker-node/issues/340#issuecomment-321669029
+RUN set -ex && \
+    apt-get update && \
+    apt-get install -y curl bash git ca-certificates python python-pip make rake g++ apt-transport-https ca-certificates bc && \
+    for key in \
+      58118E89F3A912897C070ADBF76221572C52609D \
+    ; do \
+      apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" || \
+      apt-key adv --keyserver hkp://keyserver.pgp.com:80 --recv-keys "$key" || \
+      apt-key adv --keyserver hkp://ipv4.pool.sks-keyservers.net:80 --recv-keys "$key" ; \
+    done && \
     echo "deb https://apt.dockerproject.org/repo debian-jessie main" > /etc/apt/sources.list.d/docker.list && \
     apt-get update && \
     apt-get install -y docker-engine && \
+    pip install awscli && \
     curl --create-dirs -sSLo ${JENKINS_AGENT}/agent.jar https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/${AGENT_VERSION}/remoting-${AGENT_VERSION}.jar && \
     chmod 755 ${JENKINS_AGENT} && \
     chmod 644 ${JENKINS_AGENT}/agent.jar && \
     mkdir -p ${JENKINS_HOME} && \
     useradd --home ${JENKINS_HOME} --system jenkins && \
     chown -R jenkins ${JENKINS_HOME} && \
     chmod 755 /usr/local/bin/jenkins-agent && \
-    apt-get clean
-
-WORKDIR ${JENKINS_HOME}
-USER jenkins
-
-RUN curl -L https://raw.githubusercontent.com/creationix/nvm/${NVM_VERSION}/install.sh | /bin/bash
-
-ENV SBT_VERSION=0.13.13 \
-    SBT_HOME=/usr/local/sbt \
-    SCALA_VERSION=2.11.8
-ENV PATH=${SBT_HOME}/bin:${PATH}
-
-USER root
-
-RUN curl -sL /tmp/sbt-${SBT_VERSION}.tgz "https://dl.bintray.com/sbt/native-packages/sbt/${SBT_VERSION}/sbt-${SBT_VERSION}.tgz" | \
-    gunzip | tar -x -C /usr/local && \
-    [ "0.13.13" = "${SBT_VERSION}" ] && mv /usr/local/sbt-launcher-packaging-${SBT_VERSION} /usr/local/sbt
+    apt-get clean && \
+    curl -sL /tmp/sbt-${SBT_VERSION}.tgz "https://github.com/sbt/sbt/releases/download/v${SBT_VERSION}/sbt-${SBT_VERSION}.tgz" | \
+    gunzip | tar -x -C /usr/local
 
 USER jenkins
 
-RUN printf "set scalaVersion := \"${SCALA_VERSION}\"\nupdate-sbt-classifiers\nsbtVersion\n" | sbt && \
-    rm -rf $JENKINS_HOME/project/ $JENKINS_HOME/target/
+RUN curl -sL https://raw.githubusercontent.com/creationix/nvm/${NVM_VERSION}/install.sh | /bin/bash
 
 RUN git config --global user.email "dev+jenkins@dwolla.com" && \
     git config --global user.name "Jenkins Build Agent"
 
+RUN cd $JENKINS_HOME/fake-project && \
+    echo sbt.version=0.13.16 > project/build.properties && \
```

Comment:
Should this be `1.0.4`?  Should it use the ENV variable `SBT_VERSION`?

Reply:
No, it's intentionally pulling both 1.0.4 and 0.13.16 so both are available to jobs without having to be downloaded at runtime.

Reply:
Thanks for the reminder.  I think I knew this at one point, but I forgot.